src/Modules.js

```diff
@@ -1,50 +1,51 @@
 /*
-* Core modules and their info files
-*/
+ * Core modules and their info files
+ */
 module.exports = {
-  'add-qr': require('./modules/AddQR'),
-  'average': require('./modules/Average'),
-  'blend': require('./modules/Blend'),
-  'blob-analysis': require('./modules/BlobAnalysis'),
-  'blur': require('./modules/Blur'),
-  'brightness': require('./modules/Brightness'),
-  'canvas-resize': require('./modules/CanvasResize'),
-  'channel': require('./modules/Channel'),
-  'colorbar': require('./modules/Colorbar'),
-  'color-temperature': require('./modules/ColorTemperature'),
-  'colormap': require('./modules/Colormap'),
-  'constrained-crop': require('./modules/ConstrainedCrop'),
-  'contrast': require('./modules/Contrast'),
-  'convolution': require('./modules/Convolution'),
-  'crop': require('./modules/Crop'),
-  'decode-qr': require('./modules/DecodeQr'),
-  'dither': require('./modules/Dither'),
-  'draw-rectangle': require('./modules/DrawRectangle'),
-  'dynamic': require('./modules/Dynamic'),
-  'edge-detect': require('./modules/EdgeDetect'),
-  'exposure': require('./modules/Exposure'),
-  'flip-image': require('./modules/FlipImage'),
-  'fisheye-gl': require('./modules/FisheyeGl'),
-  'histogram': require('./modules/Histogram'),
-  'gamma-correction': require('./modules/GammaCorrection'),
-  'gradient': require('./modules/Gradient'),
-  'grid-overlay': require('./modules/GridOverlay'),
-  'import-image': require('./modules/ImportImage'),
-  'minify-image': require('./modules/MinifyImage'),
```

Comment:
There is no need to change all single quotes to double. Just change one line i.e add 'compress': require('./modules/Compress'),

Reply:
I will fix that. Prettier extension on my VSCode did that. I will keep that in my mind not to make unnecessary changes.